resources/repository.rb

```diff
@@ -53,6 +55,8 @@ def initialize(*args)
 attribute :key, kind_of: String, default: nil
 attribute :key_proxy, kind_of: String, default: node['apt']['key_proxy']
 attribute :cookbook, kind_of: String, default: nil
+attribute :keyserver_retries, Integer, default: node['apt']['keyserver_retries']
```

Comment:
We shouldn't control the default with attributes here. Attributes controlling custom resources is a consuming from a consumer standpoint. I realize we're already doing that here, but we should avoid it. Just set a nice default.